doc/dev/slitmask_ids.rst

```diff
@@ -11,21 +11,30 @@ Version History
 =========   =================== =========== ===========
 *Version*   *Author*            *Date*      ``PypeIt``
 =========   =================== =========== ===========
-1.0         Debora Pelliccia    9 Oct 2020  1.1.2dev
-1.1         Debora Pelliccia    5 Nov 2020  1.2.1dev
+1.0         Debora Pelliccia     9 Oct 2020 1.1.2dev
+1.1         Debora Pelliccia     5 Nov 2020 1.2.1dev
 1.2         Debora Pelliccia    26 Jan 2021 1.3.1dev
 1.3         Debora Pelliccia    21 Oct 2021 1.6.1dev
 1.4         J. Xavier Prochaska 11 Jan 2022 1.7.1dev
 1.5         Kyle Westfall       23 Mar 2023 1.12.2dev
+1.6         Debora Pelliccia     6 Sep 2023 1.13.1dev
 =========   =================== =========== ===========
 
 ----
 
+.. _slitmask_info_instruments:
+
+Slitmask information Spectrographs
+----------------------------------
+- :doc:`../spectrographs/deimos`
+- :doc:`../spectrographs/mosfire`
+- :doc:`../spectrographs/lris` (limited)
+
 Basics
 ------
 
 The procedure used to assign slitmask ID to each slit is currently available for 
-Keck/DEIMOS, Keck/MOSFIRE, and Keck/LRIS (limited) only,
+these :ref:`slitmask_info_instruments` only,
```

Comment:
just checking that GMOS is listed here, even though our
approach is still a bit of a hack..

Reply:
Good point. I can add it.